config/default.yml

```diff
@@ -37,6 +37,14 @@ Rails/ActiveRecordAliases:
   VersionAdded: '0.53'
   SafeAutoCorrect: false
 
+Rails/ActiveRecordCallbacksOrder:
+  Description: 'Order callback declarations in the order, in which they will be executed.'
```

Comment:
I realise this is copied from the Guide, but there shouldn't be a comma after 'order'.

(I've opened https://github.com/rubocop-hq/rails-style-guide/pull/264 to fix the Guide).